README.markdown

```diff
@@ -72,6 +64,10 @@ _.mixin(s.exports());
 But it's not recommended since some of the are dropped because they collide
 with the functions already defined by Underscore.js.
 
+## Download
+
+  * [Development version](https://raw.github.com/epeli/underscore.string/master/dist/underscore.string.js) *Uncompressed with Comments*
+  * [Production version](https://github.com/epeli/underscore.string/raw/master/dist/underscore.string.min.js) *Minified*
```

Comment:
In my opinion we could add a note like in the lodash readme:

> CDN copies are available on [cdnjs](http://cdnjs.com/libraries/underscore.string/) & [jsDelivr](http://www.jsdelivr.com/#!underscore.string).

But cdnjs is not in sync, so maybe just a link to jsDelivr.